packer/install-dependencies.sh

```diff
@@ -400,9 +400,11 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
       HYPERKUBE_URL="mcr.microsoft.com/k8s/azurestack/core/hyperkube-amd64:v${KUBERNETES_VERSION}"
     else
       HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
-      CONTAINER_IMAGE="k8s.gcr.io/cloud-controller-manager-amd64:v${KUBERNETES_VERSION}"
-      pullContainerImage "docker" ${CONTAINER_IMAGE}
-      echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+      if (( $(echo ${KUBERNETES_VERSION} | cut -d"." -f2) < 16 )); then
+        CCM_IMAGE="k8s.gcr.io/cloud-controller-manager-amd64:v${KUBERNETES_VERSION}"
```

Comment:
any reason for the variable rename? the rest of the file is using `CONTAINER_IMAGE` everywhere so this is breaking the pattern (and we can't search and replace anymore). Arguably it's better to have specific variables everywhere and not reuse the same var but then we should make that change for all the components otherwise it's inconsistent.

Reply:
I was having trouble reading the code, but sure, we can do that change independently of this PR and for all enumeration blocks distinctly.